chemprop/cli/common.py

```diff
@@ -75,13 +75,14 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
     )
     featurization_args.add_argument(
         "--features-path",
-        type=list[str],  # TODO: why is this a list[str] instead of str?
+        type=Path(),  # TODO: compare with PR #635
         help="Path(s) to features to use in FNN (instead of features_generator).",
```

Comment:
```suggestion
        help="Path to features to use in FNN (instead of features_generator).",
```

We don't take multiple paths

Reply:
You change this argument in #635. I'll just remove my changes to this argument so it doesn't cause a merge conflict.